contributing/data-model-schema.txt

```diff
@@ -29,9 +29,32 @@ wastes your time. From the top-level :file:`bioformats` folder, while
 the most thorough is :command:`git clean -dfx`.
 
 
+Major or minor release?
+^^^^^^^^^^^^^^^^^^^^^^^
+
+A minor release of the OME model schema may suffice for changes like
```

Comment:
Oh dear, didn't mean to open a can of worms :) Having this sentence as part of the introduction of the documentation would certainly be a sensible option especially if we reuse them more. Happy to use @hflynn's Trello card to channel this discussion.